opendatahub-io/odh-dashboard
# Add mandatory AWS fields for model serving

## Description

Fix #804 

## How Has This Been Tested?



1. Create a new DSG project
2. Add a new Data Connection
3. Check both `S3_ENDPOINT` and `AWS_S3_BUCKET = 'AWS_S3_BUCKET` are mandatory.

## Merge criteria:
